install-mac-cc.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Do you think this file could be hidden in some `utils/` folder?

Reply:
Currently the script is dependent on which directory you're in when you run it, as it's designed to be run while you're in `hyped-2020` (uses relative pathnames). I would prefer using absolute pathnames, but I'm not entirely sure how to do this in a script. I'm a bit worried that if it was in a `utils/` directory people might run `./utils/install-mac-cc.sh` or `cd` into `utils/` first and then run `./install-mac-cc.sh`, and that would result in different things.

Reply:
Never mind, [this answer on SO](https://stackoverflow.com/a/246128/7162381) seems to do the trick in finding an absolute pathname of the script, will refactor and move it to `utils/`.